rust/parquet-bq-scripts/table_items_create.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE `{}`
+(
+  key STRING,
+  txn_version INT64,
+  write_set_change_index INT64,
+  transaction_block_height INT64,
+  table_handle STRING,
+  decoded_key STRING, -- json
+  decoded_value STRING, -- json
```

Comment:
can we also add type for _key and _value

Reply:
sync offline: will follow-up later